### PR TITLE
ilca.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/ilca.ru.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!--
+ilca.ru mismatch
+www.ilca.ru mismatch
```
Comment (J0WI) [inline src/chrome/content/rules/ilca.ru.xml]: support. => mismatch

File: src/chrome/content/rules/ilca.ru.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+bill.ilca.ru different content
+-->
+<ruleset name="ilca.ru (partial)">
+	<target host="forum.ilca.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/ilca.ru.xml]: mixed conetnet

**Comment by galeksandrp** [comment]
Wrong content on http://bill.ilca.ru. Fixed.

**Comment by J0WI** [comment]
Can't confirm. https redirects me to /user with login mask, plain http shows webpage.

**Comment by galeksandrp** [comment]
Looks like (www.)?-identical webpage on plain http is misconfiguration, because (www.)? website contains links only to https://bill.ilca.ru, which i think is for credintals-protected billing, not for exact copy of main site.

**Comment by J0WI** [comment]
This seems reasonable to me.

**Pull request merged**